package.json

```diff
@@ -38,6 +38,7 @@
     "react-intercom": "^1.0.15",
     "react-markdown": "^3.3.2",
     "react-modal": "^3.12.1",
+    "react-player": "^2.16.0",
```

Comment:
I also saw `video-react`, _also_ maintained by same firm Mux. Curious if any particular reason you chose one over the other?